secure-staff-services.md

```diff
@@ -0,0 +1,72 @@
+# Securing staff facing services
+
+## Context 
+
+We have a need to make services securely available to both our editorial and engineering staff. In the past we have relied on IP restricting access to internal IP ranges accessible on premises and via VPN by [AWS Security Groups](https://docs.aws.amazon.com/managedservices/latest/userguide/about-security-groups.html).
+
+Restricting access to IP ranges as a method of protecting our tools became impractical after the [network incident at the start of 2023](https://www.theguardian.com/media/2022/dec/21/guardian-hit-by-serious-it-incident-believed-to-be-ransomware-attack). 
+
+In addition the security offered by IP restriction relies on trusting all access from within the network which is less secure than authenticating each users identity with a trusted identity provider.
```

Comment:
Do we wish to reference this as a shift towards zero trust? It maybe use to introduce that concept to developers if they are not aware already.

Reply:
I've added some language and a link out to wiki to explain the term.